src/evict/evict_conn.c

```diff
@@ -178,7 +178,9 @@ __evict_validate_config(WT_SESSION_IMPL *session, const char *cfg[])
 
```

Comment:
[__wt_evict_file_exclusive_on](https://github.com/wiredtiger/wiredtiger/blob/ba820bef918db22d67c2f2f75108b5f3132fb8e3/src/evict/evict_lru.c#L936) is one example since I'm currently looking at it for a help ticket. 
Instead of 
"Get exclusive eviction access"
we can be more explicit and say
"Disable eviction on a file so the caller has exclusive access"

Reply:
The idea is to update the description (function comment) all the 32 functions as part of this PR. I started with evict_conn.c to gather enough feedback and apply the same writing pattern to other functions.